### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/ma
 
 * [ASP.NET Core](./src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
 * [Grpc.Net.Client](./src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md)
-* [HTTP clients](./src/OpenTelemetry.Instrumentation.Http/README.md)
+* [HTTP client](./src/OpenTelemetry.Instrumentation.Http/README.md)
```
Comment (reyang) [inline README.md:55]: I don't have strong opinion on this, the actual instrumentation lib seems to support two clients?

**Review by reyang — APPROVED** [review]
I'm okay with either way, up to you @utpilla

File: README.md (hunk 1 above)
Comment (utpilla) [reply README.md:55]: In that case, we could be more specific:

```suggestion
* [HttpClient and HttpWebRequest](./src/OpenTelemetry.Instrumentation.Http/README.md)
```

Comment (utpilla) [reply README.md:55]: Does that look good?

**Review by vishweshbankwar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (reyang) [reply README.md]: I'm slightlying leaning towards full name (System.Net.Http.HttpClient, System.Net.HttpWebRequest) - it's the same as what we did for gRPC client. No strong opinion though.

Comment (reyang) [reply README.md]: Probably change the next line (SQL client) as well.

**Pull request merged**